### PR TITLE
Assignment-4

### PR DESCRIPTION
**Pull request by shreyagoyal30**

### CODE REVIEW

File: Assignment-4/src/Team-app.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    return <div className="team-detail-container">
+      {TeamData.map((member)=>{
+       return (<CardComponent members={member} key={member.id}/>
+      );
+  })
+  };
+      </div>
+  };
+            const BodyComponent = () =>(
+
```
Comment (ankita1696) [inline Assignment-4/src/Team-app.js:28]: you could use prettier to format the code.

File: Assignment-4/src/Card-component.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+const CardComponent=(props)=>{
```
Comment (Swati27) [inline Assignment-4/src/Card-component.js:1]: You can destructure members here only..
Like -

`const CardComponent=({members})=>{`

This way you don't have to write extra line.

File: Assignment-4/src/Team-member.js [hunk 3]
```diff
@@ -0,0 +1,84 @@
+const TeamData = [
```
Comment (Swati27) [inline Assignment-4/src/Team-member.js:1]: We can have a practice to make it a json file.
It makes it easier for us to write test cases in future.
Its a good practice.

File: Assignment-4/src/index.html [hunk 4]
```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en">
+    <link rel="stylesheet" href="style.css">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
```
Comment (Swati27) [inline Assignment-4/src/index.html:8]: Give it a relevent name. May be like- Assignment 4

File: Assignment-4/src/Card-component.js (hunk 2 above)
Comment (shreyagoyal30) [reply Assignment-4/src/Card-component.js:1]: Thanks,It is a very good idea.

**Pull request merged**